src/pages/Goals/index.js

```diff
@@ -1,45 +1,48 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 import { Link } from 'react-router-dom';
+import Header from '../../components/Header';
 import Goal from './Goal';
```

Comment:
Hi @danielufeli, 
Following the Figma design, the button `Get started` on the Goals page should be hidden. The same thing for the questions page.